css/components/modal/_modal-body.scss

```diff
@@ -0,0 +1,159 @@
+.modal-body {
+  display: inline-block;
+  height: auto;
+  overflow-y: scroll;
+  position: relative;
+  width: 100%;
+
+  .wp-editor-area {
+    resize: vertical;
+  }
+
+  .wp-editor-wrap,
+  .wp-editor-container,
+  .wp-editor-container
+  .wp-editor-area {
+    height: 100%;
+  }
+
+  >form {
```

Comment:
This form seems quite specific. Should we extract it in its own component (like `form-questions`)?